frontend/src/components/Radio/Radio.tsx

```diff
@@ -295,6 +305,8 @@ export const OthersInput = forwardRef<InputProps, 'input'>(
         ref={mergedInputRef}
         {...props}
         onChange={handleInputChange}
+        // This is to enhance radio a11y: https://github.com/opengovsg/FormSG/pull/4925
```

Comment:
we should not link to any PRs, we should add a specific comment. URLs may die

Reply:
ammended!